commands/qemu-guest-tools/actions_test.go

```diff
@@ -0,0 +1,261 @@
+package qemuguesttools
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	goruntime "runtime"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
```

Comment:
hrm, we should be consistent in the team how we do this. Currently I rely on `gofmt` through [vim-go](https://github.com/fatih/vim-go) for code formatting. I will check how I can change it to use `goimports`.